docs/en/api-reference/image-generation.md

```
@@ -2,231 +2,167 @@
-## Using Custom Image Tools
+## Request Parameters
 
-Alternatively, you can generate images by using the `image_generation` tool in your request. This approach provides more control over image generation parameters.
-
-### Tool-based Image Generation
-
-To use custom image tools, include an `image_generation` tool in the `tools` array of your request:
-
-### Example
+| Parameter | Type | Description | Default |
+|-----------|------|-------------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)